googlemock/include/gmock/internal/gmock-SignatureOf.h

```diff
@@ -0,0 +1,38 @@
+#ifndef THIRD_PARTY_GOOGLETEST_GOOGLEMOCK_INCLUDE_GMOCK_SIGNATUREOF_H_
```

Comment:
We don't need or want one file for each little helper API.
Keep it in the same file, just have it in the internal namespace.

Reply:
Sure, I changed this to be your way!